src/Table.tsx

```diff
@@ -129,9 +129,9 @@ export interface TableProps<RecordType = unknown>
   rowClassName?: string | RowClassName<RecordType>;
 
   // Additional Part
-  title?: PanelRender<RecordType>;
   footer?: PanelRender<RecordType>;
   summary?: (data: readonly RecordType[]) => React.ReactNode;
+  caption?: string | React.ReactNode;
```

Comment:
> Or we keep title prop without adding caption but replace dom node by <caption /> in current implementation.

Is it a better proposal?

Reply:
If we want to avoid a breaking change, refactoring how the existing `title` prop gets used (inside a `<caption>`) would be the better option.  If we want to potentially avoid confusing consumers who are expecting a `caption` prop rather than a `title` prop, the first option would be better. I just pushed a commit deprecating `title`, but let me know if you want to go with the other option.